tests/Application/package.json.~1.11.0.dist

```diff
@@ -26,11 +26,12 @@
         "sass": "^1.50.0",
         "sass-loader": "^7.0.1",
         "upath": "^1.1.0",
+        "webpack": "^5.76.1",
         "yargs": "^6.4.0"
     },
     "scripts": {
-        "build": "gulp build",
-        "gulp": "gulp build",
+        "build": "encore production",
+        "dev": "encore dev",
```

Comment:
Changing the build system from gulp to encore has to be mentioned in README.md file.

Reply:
But thank to this change in scripts section nothing change from documentation point of view 

$ composer install
$ cd tests/Application
$ yarn install
$ yarn build // would have to be changed to yarn encore dev i i wouldnt change script section
$ bin/console assets:install public -e test
$ bin/console doctrine:database:create -e test
$ bin/console doctrine:schema:create -e test
$ bin/console server:run 127.0.0.1:8080 -d public -e test
$ sudo service elasticsearch start

Reply:
:ok_hand: